lib/spack/spack/test/cmd/compiler.py

```diff
@@ -182,6 +184,7 @@ def clangdir(tmpdir):
     yield tmpdir
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Error on Win")
```

Comment:
FWIW it's easy in this case to explain what is going wrong: the `clangdir` fixture creates bash scripts, which won't run on Windows (and likewise for the other tests that fail in this module).

Reply:
Amended.